pkg/gds/config/config.go

```diff
@@ -10,16 +10,17 @@ import (
 	"github.com/kelseyhightower/envconfig"
 	"github.com/rs/zerolog"
 	"github.com/trisacrypto/directory/pkg/sectigo"
+	"github.com/trisacrypto/directory/pkg/utils/logger"
 )
 
 // Config uses envconfig to load required settings from the environment and validate
 // them in preparation for running the TRISA Global Directory Service.
 type Config struct {
-	DirectoryID string          `split_words:"true" default:"vaspdirectory.net"`
-	SecretKey   string          `split_words:"true" required:"true"`
-	Maintenance bool            `split_words:"true" default:"false"`
-	LogLevel    LogLevelDecoder `split_words:"true" default:"info"`
```

Comment:
Moved the `LogLevelDecoder` to the `pkg/utils/logger` for use in both GDS and TRTL

Reply:
Good idea!